blocks/aem-asset-selector/aem-asset-selector-util.js

```diff
@@ -378,6 +378,56 @@ export async function renderAssetSelectorWithImsFlow(cfg) {
     hideTreeNav: true,
     runningInUnifiedShell: false,
     noWrap: true,
+    filterSchema: [
+      {
+        header: 'File Type',
+        groupKey: 'TopGroup',
+        fields: [
+          {
+            element: 'checkbox',
+            name: 'type',
+            defaultValue: ['image/*'],
+            readOnly: true,
+            options: [
+              {
+                label: 'Images',
+                value: 'image/*',
+              },
+            ],
+          },
+        ],
+      },
+      {
+        fields:
+              [
+                {
+                  element: 'checkbox',
+                  name: 'type',
+                  options: [
+                    {
+                      label: 'JPG',
+                      value: 'image/jpeg',
+                    },
+                    {
+                      label: 'PNG',
+                      value: 'image/png',
+                    },
+                    {
+                      label: 'TIFF',
+                      value: 'image/tiff',
+                    },
+                    {
+                      label: 'GIF',
+                      value: 'image/gif',
```

Comment:
We can also add webp here once we've tested it works with getting high res assets (canvas supports it, we just didn't have the assets to test and I don't have enough permissions to add assets...).